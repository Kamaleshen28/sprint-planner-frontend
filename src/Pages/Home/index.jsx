import React from 'react';

import { Topbar, Title, InputForm } from '../../Components';

function Home() {
  return (
    <>
      <Topbar />
      <Title />
      <InputForm />
    </>
  );
}

export default Home;
