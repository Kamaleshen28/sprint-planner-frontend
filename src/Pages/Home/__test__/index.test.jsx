import { render } from '@testing-library/react';
import React from 'react';

import Home from '..';

describe('Home', () => {
  it('should render successfully', () => {
    const screen = render(<Home />);
    expect(screen.asFragment()).toMatchSnapshot();
  });
});
