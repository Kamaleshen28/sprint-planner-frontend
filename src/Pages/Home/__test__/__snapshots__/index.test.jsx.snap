// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render successfully 1`] = `
<DocumentFragment>
  <div
    class="home"
  >
    <h1>
      Home
    </h1>
  </div>
</DocumentFragment>
`;
